### PR TITLE
Simplifying and removing data.json

### PR DESCRIPTION
**Pull request by glennblock**

### CODE REVIEW

File: content/src/policies/__id/delete.rego [hunk 1]
```diff
@@ -1,25 +1,38 @@
 package peoplefinder.DELETE.api.users.__id
 
-import input.policy.path
+# Policy for deleting an employee in PeopleFinder
+#
+# Note: This policy is for illustration. The Delete button visibility is used by PeopleFinder, but the button does not call the delete API.
```
Comment (ogazitt) [inline content/src/policies/__id/delete.rego]: I think this is an app issue, not a policy issue, so it should be called out in the app. E.g. we can modify the app to actually call the delete API, and get back a message "This is a sample, so no user was actually harmed :)"

File: content/src/policies/__id/put.rego [hunk 2]
```diff
@@ -1,33 +1,29 @@
...
-default enabled = false
-
+# Iterate over all the user's roles and return "true" if "admin" is present
 allowed {
-	some index
-	data.roles[user_roles[index]].perms[path].allowed
+	"admin" = user_roles[_]
 }
 
+# Check if the user is the same as the employee being updated
```
Comment (ogazitt) [inline content/src/policies/__id/put.rego]: `# allowed is also true if the logged-in user is the same as the employee being updated`

**Review by ogazitt — CHANGES_REQUESTED** [review]

File: content/src/policies/__id/delete.rego (hunk 1 above)
Comment (glennblock) [reply content/src/policies/__id/delete.rego]: Fixed

File: content/src/policies/__id/put.rego (hunk 2 above)
Comment (glennblock) [reply content/src/policies/__id/put.rego]: Updated

**Review by ogazitt — APPROVED** [review]

**Review by ogazitt — APPROVED** [review]

**Pull request merged**